src/cls/IPM/Main.cls

```diff
@@ -139,6 +141,28 @@ This command is an alias for `module-action module-name publish`
 <modifier name="use-external-name" aliases="use-ext" dataAlias="UseExternalName" dataValue="1" description="Publish the package under the &lt;ExternalName&gt; of the package. If ExternalName is unspecified or illegal, an error will be thrown."/>
 </command>
 
+<command name="update" dataPrefix="D">
+<summary>Updates a module to a newer version.</summary>
+<description>
+	Updates a module to a newer version. Also runs all of the update steps from the current version to the newer version.
+</description>
+
+<!-- Examples -->
+<example description="Updates the HS.JSON module from the current version to version 3.0.0.">
+	update HS.JSON 3.0.0
+</example>
+
+<!-- Parameters -->
+<parameter name="module" required="true" description="Name of module on which to perform update actions" />
+<parameter name="version" description="Version (or version expression) of module to update; defaults to the latest available if unspecified." />
```

Comment:
Should also specify that this is ignored if `-path` is provided since it will then update to that version